docs/developers/developing_iree/sanitizers.md

```diff
@@ -1,79 +1,49 @@
-# Using Address/Memory/Thread Sanitizers
+# Using Sanitizers (ASAN, TSAN, etc)
 
-[AddressSanitizer](https://clang.llvm.org/docs/AddressSanitizer.html),
-[MemorySanitizer](https://clang.llvm.org/docs/MemorySanitizer.html) and
-[ThreadSanitizer](https://clang.llvm.org/docs/ThreadSanitizer.html) are tools
-provided by `clang` to detect certain classes of errors in C/C++ programs. They
-consist of compiler instrumentation (so your program's executable code is
-modified) and runtime libraries (so e.g. the `malloc` function may get
-replaced).
+[AddressSanitizer](https://clang.llvm.org/docs/AddressSanitizer.html) and
+[ThreadSanitizer](https://clang.llvm.org/docs/ThreadSanitizer.html), abbreviated as "ASAN" and "TSAN" respectively, are LLVM instrumentation features helping to detect certain classes of bugs. They are available both via Clang (to build C/C++ code, such as IREE's binaries) and via `iree-compile` (to build IREE bytecode modules). This page discusses both Clang and `iree-compile` usage, and the interactions between the two, since there are consistency requirements between how IREE bytecode modules are instrumented and how the IREE runtime, which loads them, is instrumented.
 
-They are abbreviated as "ASan", "MSan" and "TSan" respectively.
```

Comment:
Keep these capitalizations? https://github.com/google/sanitizers/wiki/AddressSanitizer and https://clang.llvm.org/docs/AddressSanitizer.html both use `ASan` and not `ASAN`. (Yeah, it's weird... took me a while to get used to the mixed casing)